### DISCUSSION TITLE
ManimGL- is not using GPU

### DISCUSSION THREAD

**Discussion by Srikar-Ganta** [3b1b/manim#2118] [Q&A]
I have run the animation with manimCE, then it took me around 22secs.
when I run same code with manimGL, then it took me around 36secs.

I have seen GPU utilisation in task manager also, I couldn't see much difference in utilisation also. 
Am i missing anything here? should we cofigure anything to for GPU to work.

**Comment by jake-downey** [comment]
Same is happening for me.  ManimGL is working correctly, but it doesnt render using my GPU.  My usage goes to 100% CPU usage and virtually no GPU usage.  My GPU is a Radeon Pro 580 8 GB, it's the onboard chipset for my iMac and gets utilized in other rendering situations but seemingly not with opengl when using manimgl.
